uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/addon/EventFilter.java

```diff
@@ -197,6 +202,17 @@ public Builder withRetain(Boolean retain) {
             return this;
         }
 
+        /**
+         * Sets user properties.
```

Comment:
Sets or resets

Reply:
It's builder's method, that's why I wrote sets.